src/assets/css/themes/_variables.darkly.scss

```diff
@@ -1,35 +1,51 @@
-$white: #fff;
-$gray-100: #f8f9fa;
+@import "../../../../node_modules/bootstrap-v4/scss/functions";
+@import "../../../../node_modules/bootstrap-v4/scss/variables";
+
+// Colors
 $gray-200: #ebebeb;
-$gray-300: #dee2e6;
-$gray-400: #ced4da;
-$gray-500: #adb5bd;
 $gray-600: #888;
```

Comment:
Overwriting variables _after_ importing the Bootstrap variables might not work, see [Theming Bootstrap: Variable defaults](https://getbootstrap.com/docs/4.1/getting-started/theming/#variable-defaults)

> Variable overrides within the same Sass file can come before or after the default variables. However, when overriding across Sass files, your overrides must come before you import Bootstrap’s Sass files.

Reply:
This is fine, because we are importing them again later in the main `darkly.scss` file.

We need them to set some of our theme variables to existing Bootstrap color variables; then we import Bootstrap again later.

Reply:
I'm incorrect! This is indeed causing issues. I've fixed this in the latest commit.